examples/bri-3/package.json

```diff
@@ -25,6 +25,10 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@automapper/classes": "^8.7.6",
```

Comment:
can we now remove all ttsc stuff, all changes related to packages, scripts, ts config and readme?

Reply:
will be done in a separate PR